ingestion/src/metadata/profiler/metrics/static/stddev.py

```diff
@@ -95,7 +95,14 @@ def df_fn(self, dfs=None):
         import pandas as pd  # pylint: disable=import-outside-toplevel
 
         if is_quantifiable(self.col.type):
-            return pd.concat(df[self.col.name] for df in dfs).std()
+            try:
+                return pd.concat(df[self.col.name] for df in dfs).std()
+            except MemoryError:
+                logger.error(
+                    f"Unable to compute distinctCount for {self.col.name} due to memory constraints."
+                    f"We recommend using a smaller sample size or partitionning."
+                )
+                return 0
```

Comment:
let's return None otherwise we will be setting the median value to 0 when we did not compute it at all.

Reply:
resolved